tapiro/core/src/main/scala/io/buildo/tapiro/Meta.scala

```diff
@@ -9,33 +9,42 @@ import cats.data.NonEmptyList
 object Meta {
   val codecsImplicits = (routes: List[TapiroRoute]) => {
```

Comment:
This function is getting quite messy...

Part of the problem is that we're delaying the conversion to scalameta type for each type of codec because we want to call `.distinct` while we still have a list of metarpheus types, because it wouldn't work on scalameta types because `==` is by reference

Maybe we could use structural equality from scala.meta.contrib (https://scalameta.org/docs/trees/guide.html#compare-trees-for-equality) to clean this up a bit?

Reply:
I think using `isEqual` is a good idea

Reply:
I've tried to rewrite that function and I think it reads a bit better now